docs/getting-started.md

```diff
@@ -212,23 +212,23 @@ resources inspired by [paulbouwer](https://github.com/paulbouwer)'s
     import { App, Chart } from 'cdk8s';
 
     // imported constructs
-    import { KubeDeployment, KubeService, IntOrString } from './imports/k8s';
+    import { Deployment, Service, IntOrString } from './imports/k8s';
```

Comment:
The latest version actually has the `Kube` prefix for all of these. Sure you tested with the latest import?

Reply:
Great point, I was using `0.33.0`, seems to be fixed in latest version. Though also noticing now that latest version is not publishing to Brew... so I'll make a different issue for that

Reply:
```suggestion
    import { KubeDeployment, KubeService, IntOrString } from './imports/k8s';
```